src/ui/js/UserPrefs.js

```diff
@@ -279,16 +285,22 @@ UserPrefs.actionSetPrefs = function() {
     },
   };
 
-  var browserBlurb = 'These preferences will only apply to the browser ' +
-    'you\'re currently using.';
-  var browserPrefs = {
+  var matBlurb = 'These preferences control the look and feel of the ' +
+    'game mat.';
+  var matPrefs = {
+    'die_background_select': {
+      'text': 'Die background',
+      'type': 'select',
+      'value': Api.user_prefs.die_background,
+      'source': dieBackgroundDefaults,
+    },
     'noImages': {
-      'text': 'Don\'t load button or player images',
+      'text': '(only this browser) Don\'t load button or player images',
```

Comment:
Hmm, is it clear enough what this means?  I think i'd say: "Don't load button or player images (only applies to the current browser)", and likewise for compact mode.